Packs/RecordedFuture/ReleaseNotes/1_6_0.md

```diff
@@ -21,9 +20,10 @@
 
 - Updated the Docker image to: *demisto/python3:3.10.11.61265*.
 
```

Comment:
please revert all changes at 1_5_0.md, 1_6_0.md , 1_16_1.md and add them to 1_6_3.md

Reply:
done